token-tracker/src/lib/uniswap.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios"
+
+const UNISWAP_API_URL =
+  "https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2"
+
+export const getPairData = async (pairId) => {
+  const response = await axios.post(UNISWAP_API_URL, {
+    query: `query pairquery {
+        pair(id: "${pairId}") {
+            token0 {
+                symbol,
+                derivedETH,
+            },
+            token1 {
+                symbol,
+                derivedETH,
+            },
+            reserve0,
+            reserve1,
+            reserveETH,
+            reserveUSD,
+            token0Price,
+            token1Price,
+            totalSupply,
```

Comment:
Do we need to fetch all these properties?

Reply:
This is gone now 09609e6